macOS/DuckDuckGo/Menus/MainMenu.swift

```diff
@@ -647,53 +647,53 @@ final class MainMenu: NSMenu {
 
     // MARK: - Debug
 
-    let internalUserItem = NSMenuItem(title: "Set Internal User State", action: #selector(MainViewController.internalUserState))
+    let internalUserItem = NSMenuItem(title: "Set Internal User State", action: #selector(AppDelegate.internalUserState))
```

Comment:
make the menu items enabled when no windows are open by moving the methods to the AppDelegate